parcels/field.py

```diff
@@ -976,6 +977,34 @@ def computeTimeChunk(self, data, tindex):
 
         return data
 
+    def __add__(self, field):
+        if isinstance(self, Field) and isinstance(field, Field):
+            return SummedField([self, field])
+        elif isinstance(field, SummedField):
+            return field.__add__(self)
+
+
+class SummedField(list):
```

Comment:
Could you add some doc here?
Especially advert that even now that we have sums, we still keep the info on the different sum terms 
(that we can say SummedField[4])